report/res/index.css

```diff
@@ -0,0 +1,30 @@
+body {
+    margin: 0px;
+    overflow: hidden;
+    font-family: sans-serif;
+}
+
+#placeholder {
+    width: 100%;
+    height: 100vh;
+}
+
+.report-col-count-2 .ddCell:nth-child(4n+3),
+.report-col-count-2 .ddCell:nth-child(4n+4),
+.report-col-count-3 .ddCell:nth-child(6n+6),
+.report-col-count-3 .ddCell:nth-child(6n+4),
+.report-col-count-3 .ddCell:nth-child(6n+6),
+.report-col-count-4 .ddCell:nth-child(8n+5),
+.report-col-count-4 .ddCell:nth-child(8n+6),
+.report-col-count-4 .ddCell:nth-child(8n+7),
+.report-col-count-4 .ddCell:nth-child(8n+8),
+.report-col-count-5 .ddCell:nth-child(10n+6),
+.report-col-count-5 .ddCell:nth-child(10n+7),
+.report-col-count-5 .ddCell:nth-child(10n+8),
+.report-col-count-5 .ddCell:nth-child(10n+9),
+.report-col-count-5 .ddCell:nth-child(10n+10) {
+    background:#F3F3F3;
+    -webkit-box-shadow: 0px 0px 0px 6px rgba(243,243,243,1);
+    -moz-box-shadow: 0px 0px 0px 6px rgba(243,243,243,1);
+    box-shadow: 0px 0px 0px 6px rgba(243,243,243,1);
+}
```

Comment:
What does this css do?

Reply:
Provides the background color for the gutter between grid cells to give the appearance of a solid background color alternating between white and light grey.